packages/server/src/util/memoizer-redis/index.ts

```diff
@@ -90,7 +92,10 @@ export default function memoizerFactory(client: Redis.Redis, opts: Partial<Memoi
 
     // the actual memoized function
     memoized = async function memoized(...args: Array<any>): Promise<T> {
-      const cacheKey = getCacheKey(keyPrefix, fnKey, hashArgs(args));
+      let cacheKey = getCacheKey(keyPrefix, fnKey, hashArgs(args));
+      if (typeof cacheKeyOverride === 'function') {
+        cacheKey = cacheKeyOverride(cacheKey, keyPrefix, fnKey, args);
```

Comment:
In what scenario would the override come into play?

Reply:
i dreamt about this 😂 . this idea was only valid for moving to the workers but then I found out you already exported the v2 fetcher. I'm going to just use the same fn.